prime-router/docs/api/delivery.yml

```diff
@@ -0,0 +1,144 @@
+openapi: 3.0.2
+info:
+  title: Prime ReportStream
+  description: A router of public health data from multiple senders and receivers
+  contact:
+    name: USDS at Centers for Disease Control and Prevention
+    url: https://reportstream.cdc.gov
+    email: reportstream@cdc.gov
+  version: 0.2.0-oas3
+paths:
+  /v1/receivers/{receiverFullName}/deliveries/submitters/search:
+    post:
+      description: Return a list of all the the providers, facilities and senders that have sent results to a receiver
+      security:
+        - OAuth2: [ user ]
+        - ApiKeyAuth: [ ]
+      parameters:
+        - in: query
+          name: receiverFullName
+          schema:
+            type: string
+          required: true
+          description: The full name of the receiver to fetch submitters for
+      requestBody:
+        description: The search parameters for the request
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SubmitterApiSearch'
+      responses:
+        '200':
+          description: The results from running the search
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SubmitterApiSearchResults'
+        '401':
+          description: Unauthorized
+        '400':
+            description: The search request could not be parsed
+        '404':
+          description: The receiver could not be found
+components:
+  schemas:
+    SubmitterApiSearch:
+      type: object
+      properties:
+        sort:
+          description: The sorting to apply to the results
+          type: object
+          properties:
+            direction:
+              description: The order to apply the sort property
+              type: string
+              enum:
+                - DESC
+                - ASC
+            'property':
+              description: The property to sort by
+              type: string
+              enum:
+                - id
+                - name
+                - first_report_date
+                - test_result_count
```

Comment:
test_result_count is not needed

Reply:
Yep, not needed for this view, but there is another page that leverage this API (after some more filters are added)